Dockerfile.host-check

```diff
@@ -15,4 +15,4 @@ RUN apt-get -y update && \
 
 COPY scripts/host-check .
 
-ENTRYPOINT ["scripts/host-check"]
+ENTRYPOINT ["./host-check"]
```

Comment:
As discussed, we will need a new 1.1.17 release for this as this is a bugfix we should release immediately.